### PR TITLE
Fix #99

### LINKED ISSUE #99: `harmonic run` automatically open default browser
It would save a bit of time if `harmonic run` opened `localhost:port` in the default browser.
Perhaps `harmonic run` could also take a flag to not open a browser window.

### PR DESCRIPTION
**Pull request by UltCombo**
`harmonic run` opens web browser and harmonic `new_post|page` opens the markdown files in the text editor by default (prevent it passing a `--no-open` flag), fix #99.
Details in the commit messages.

### CODE REVIEW

File: src/bin/cli/harmonic.js [hunk 1]
```diff
@@ -37,31 +37,37 @@ program
...
 program
-    .command('new_post ["title"] [path]')
+    .command('new_post <title> [path]')
+    .option('--no-open', 'Don\'t open the markdown file(s) in editor')
     .description('Create a new post')
     // [BUG] https://github.com/jshint/jshint/issues/1849 - can't use arrow function
-    .action(function(title, path = '.') {
-        newFile(path, 'post', title);
+    // [BUG] https://github.com/jshint/jshint/issues/1779#issuecomment-68985429
+    .action(function(title, path = '.', { open: autoOpen }) { // jshint ignore:line
```
Comment (UltCombo) [inline src/bin/cli/harmonic.js]: When we define `.option('--no-open', ...)`, Commander.js creates a boolean option called "open" which defaults to `true` and is only false when the `--no-open` flag is present.

Relevant source:
- https://github.com/tj/commander.js/blob/29b4ea91a18679546980ce1a1e07957d1cbf9fb5/index.js#L316-L317
- https://github.com/tj/commander.js/blob/29b4ea91a18679546980ce1a1e07957d1cbf9fb5/index.js#L356-L362

**Comment by UltCombo** [comment]
@jaydson indeed, the build is failing on Travis too.
Weird, it was passing when I first pushed those commits, something might have changed in gulp-6to5 or 6to5.

**Comment by UltCombo** [comment]
I'll debug this when I get home.

**Comment by UltCombo** [comment]
Found the issue.
The new gulp-6to5 version depends on the [6to5-core package](https://www.npmjs.org/package/6to5-core) instead of 6to5, that's why the `require` fails.
This error is new, guess this is one more reason we should use a direct dependency then.

**Comment by UltCombo** [comment]
@sebmck sorry to bother you, but what is the correct approach in this case -- we are shipping 6to5'ed code, but still need to require `6to5/polyfill` in order for certain features to work.

In this case, should we have `6to5`/`6to5-core` as a dependency of the 6to5'ed package so that it can `require('6to5/polyfill')`? Isn't there a `6to5-polyfill` or such to add as a dependency of the 6to5'ed package instead of depending on the whole transpiler?

**Comment by sebmck** [comment]
@UltCombo `6to5-core` is exactly the same as `6to5` except it lacks the CLI which results in fewer dependencies to install etc. It includes all the same files so you can require `6to5-core/polyfill` all the same. Unfortunately there's no `6to5-polyfill` package but I'm not sure if it'd be worth the extra package as it's literally only:

```javascript
require("core-js/shim");
require("regenerator/runtime");
```

**Comment by UltCombo** [comment]
@sebmck makes sense, thanks!

**Comment by sebmck** [comment]
@UltCombo No problem! Feel free to ping me in any issues as I'm more than happy to help.

**Comment by UltCombo** [comment]
@sebmck thanks again!

@jaydson please take a look. I've added core-js and regenerator as dependencies so `npm install` won't have to pull the whole 6to5 package after we publish.

**Comment by jaydson** [comment]
Working just fine :)
Thanks @UltCombo and @sebmck !

**Comment by UltCombo** [comment]
Yaay :smile:

**Pull request merged**